### DISCUSSION TITLE
My Github Pro highlight disappeared after I sponsored someone

### DISCUSSION THREAD

**Discussion by 0einstein0** [community/community#35314] [Other Feature Feedback, Questions, & Ideas]
My Github had PRO package due to the student account but when I got the sponsor badge the PRO highlight disappeared from the higlights. How do I get it back?

**Comment by fadyat** [comment]
Hello, maybe you have disabled the checkbox with the display of PRO badge?
<img width="373" alt="Screenshot 2022-10-06 at 6 37 18 PM" src="https://user-images.githubusercontent.com/70910148/194356844-ce78924d-f15c-4731-b69a-5e57d45961ce.png">

**Reply by 0einstein0** [reply]
I don't even get that option anymore.. 

<img width="421" alt="image" src="https://user-images.githubusercontent.com/51617644/194452390-06a0b735-db74-46a4-860a-45fb4edf570a.png">

**Reply by fadyat** [reply]
And you still have student GitHub package active, right?

**Reply by 0einstein0** [reply]
<img width="939" alt="image" src="https://user-images.githubusercontent.com/51617644/194495304-25b2b8f3-efb7-485f-a37a-a4464bfb96b6.png">
I assume so

**Reply by 0einstein0** [reply]
<img width="956" alt="image" src="https://user-images.githubusercontent.com/51617644/194495809-6dc6cfb1-bc3b-478f-8406-fba4a8905a89.png">

**Reply by fadyat** [reply]
idk 😢

**Reply by 0einstein0** [reply]
I added a ticket in Github support they haven't responded yet.

**Reply by fadyat** [reply]
You are just too PRO for this badge 😁
